containers/ddev-webserver/README.md

```diff
@@ -9,7 +10,7 @@ This container image is part of DDEV, and is not typically used stand-alone.
 
 * Nginx
 * Apache
-* Many PHP versions
+* All supported, non-EOL PHP versions
```

Comment:
No need to change this, but of course we support so very many EOL PHP versions. Not sure how to say it well, but "Many" was probably decent terminology. Maybe "A stupendous number of weird PHP versions" :) Again, leave it :)